tests/e2e/tests/plugins/TypescriptPlugin.spec.ts

```diff
@@ -49,19 +48,18 @@ const fileFolderPath: string = `${projectName}`;
 const debugFileFolderPath: string = `${projectName}/app`;
 const debugFile: string = 'app.js';
 const tabTitle: string = 'typescript-node-debug.ts';
-let workspaceName: string = 'typescript-plugin';
 
 suite(`The 'TypescriptPlugin and Node-debug' tests`, async () => {
     suite('Create workspace', async () => {
         test('Create workspace using factory', async () => {
             await browserTabsUtil.navigateTo(factoryUrl);
         });
 
-        projectAndFileTests.waitWorkspaceReadiness(projectName, subRootFolder);
+        workspaceHandlingTests.obtainWorkspaceNameFromStartingPage();
```

Comment:
It's strange to see code between the tests.

IMHO, it's part of `Wait until created workspace is started` test, which is missed in here.